java-devcourse-project/src/DataStructureIntro/DataStructureIntroTask1.java

```diff
@@ -0,0 +1,37 @@
+package DataStructureIntro;
+
+import java.util.Arrays;
+
+public class DataStructureIntroTask1 {
+    public static void main(String[] args) {
+        int index = 3;
+        var array = arrayInit(6);
+        System.out.println(Arrays.toString(array));
+        AddToArray(array, 6);
+        System.out.println("Value by index " + index + " : " + (getValueByIndex(array, index)));
+    }
+
+    static int[] arrayInit(int size) {
+        int[] array = new int[size];
+        for (int i = 0; i < array.length; i++) {
+            array[i] = i;
+        }
+        return array;
+    }
+
+    static void AddToArray(int array[], int newValue) {
```

Comment:
именование методов в Java всегда начинается с маленькой буквы.
```suggestion
    static void addToArray(int array[], int newValue) {
```